in_text/text_00_parent.yml

```
@@ -2,7 +2,7 @@ title: >-
   Data to support water quality modeling efforts in the Delaware River Basin
 
 abstract: >-
-  <p> This data release contains information to support water quality modeling in the Delaware River Basin (DRB).
+  This data release contains information to support water quality modeling in the Delaware River Basin (DRB).
```

Were these html tags included intentionally? When I open the xml file in the metadata wizard app and general the word doc, the tags still show up. Happy to get on the phone and show you what I mean...I wanted to see if they were intentional before I suggested removing all of them.
```suggestion
  This data release contains information to support water quality modeling in the Delaware River Basin (DRB).
```